### PR TITLE
feat: i18n en_US translation files

### PR DESCRIPTION
**Pull request by myan9**
Part of #1696

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [ ] 💅 Enhancement
- [x] 🚀 New feature

#### Please confirm that your PR fulfills these requirements

- [x] Multiple commits are squashed into one commit.
- [x] The commit message follows [Conventional Commits](https://github.com/IBM/kui/blob/master/CONTRIBUTING.md#conventional-commits), which allows us to autogenerate release notes; e.g. `fix(plugins/plugin-k8s): fixed annoying bugs`
- [x] All npm dependencies are pinned.

### CODE REVIEW

**Comment by myan9** [comment]
> lgtm

Updates: I removed the singular cases for `logs` and `containers`, since we don't need the singular case for now.

**Comment by starpit** [comment]
what about "You are using the default theme"?

**Comment by myan9** [comment]
> what about "You are using the default theme"?

Thanks! Updated

File: packages/app/i18n/locales/en_US/translation.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "today": "Today at {0}",
+  "sidecarExpand": "Expand to full width",
+  "sidecarMinimize": "Minimize the sidecar",
+  "sidecarRestore": "Restore split screen",
+  "screenshotBalloon": "Capture screenshot"
+}
```
Comment (starpit) [inline packages/app/i18n/locales/en_US/translation.json]: missing Close; also the Minimize text is now just "Minimize"

Comment (myan9) [reply packages/app/i18n/locales/en_US/translation.json]: Thanks! Updated

**Comment by starpit** [comment]
i'll merge in what we have so far, then we will continue to refine

**Pull request merged**